timeseries/src/autogluon/timeseries/learner.py

```diff
@@ -7,7 +7,7 @@
 
 from autogluon.core.learner import AbstractLearner
 from autogluon.timeseries.dataset.ts_dataframe import TimeSeriesDataFrame
-from autogluon.timeseries.evaluator import TimeSeriesEvaluator
```

Comment:
Response to the general comment by @canerturkmen 
> That said, maybe we should also remove that code in this PR for consistency.

I'm not 100% sure about that since the `TimeSeriesEvaluator` lived in this limbo between public & private API (it was available in the namespace `autogluon.timeseries`). At least, I would add deprecation warnings to the `TimeSeriesEvaluator` + turn this class into a thin wrapper around `autogluon.timeseries.metrics` for backward compatibility.

Reply:
+1 to turning it into a wrapper.